src/components/Book.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Books from './Books';
+
+const BookContent = ({ books }) => (
+  <ul className="book-content">
+    {
+books.map((book) => <Books key={book.id} book={book} />)
+}
+  </ul>
+
+);
+BookContent.propTypes = {
+  books: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.number.isRequired,
+      title: PropTypes.string.isRequired,
+      author: PropTypes.string.isRequired,
+      category: PropTypes.string.isRequired,
+    }).isRequired,
+  ).isRequired,
+};
```

Comment:
You have an error validating books. Kindly check the error and fix accordingly 
![error](https://user-images.githubusercontent.com/12239821/175367398-31b755e6-53cd-40a1-a986-f665f8695c4d.png)

Reply:
@chrystalme I have not inserted the API id that's why the error is showing

Reply:
You need to refactor your code. I have the solutions on my system if you would like to see. Your category does not have to be require since you have not implemented it